content/en/getting_started/workflow_automation/_index.md

```diff
@@ -0,0 +1,157 @@
+
+6. Test this Slack action by clicking the green **Test** icon in the **Configure** tab of the action.
+7. To view your workflow's name in the monitor's notification dropdown, save and publish your workflow. Click **Publish** from the workflow's page.
+
+## Test and publish the workflow
+
+To apply your changes, ensure that you save your workflow by clicking **Save**. Next, test out your workflow by manually triggering it: 
+
+{{< img src="/getting_started/workflow_automation/run_workflow.png" alt="Trigger a workflow manually" style="width:90%;" >}}
+
+To trigger the workflow manually, click **Run** from the workflow page, and enter the values for the trigger variables.
+
+Confirm that running the workflow creates a Jira ticket and sends a Slack message as you specified. 
+
+Scheduled and triggered workflows don't trigger automatically until you've published them. To publish the workflow, click **Publish** from the workflow's page.
+
+<div class="alert alert-info">If you are testing a workflow with an active Slack and Jira account, then actual Slack posts and Jira tickets are created.</div>
+
+## Update the monitor that triggers your workflow
+
+1. Navigate to the [Monitors page][7] in Datadog.
+2. Find the monitor you'd like to use to trigger the workflow and edit it, or create a new monitor.
+3. In the message section, add the full workflow mention name to an alert notification. The mention name should start with `@workflow-`. For example, `@workflow-Create-Jira-Ticket`
+    - You can pass trigger variables into the workflow by using a comma-separated list with the syntax `@workflow-name(key=value, key=value)`. For example, `@workflow-Create-Jira-Ticket(hostname=host.name)`
+4. Click **Test Notifications** to test the workflow and all of this monitor's notifications. 
+5. Save the monitor. 
+
+{{< img src="/getting_started/workflow_automation/monitor_trigger.png" alt="Trigger a workflow from a Monitor">}}
+
+Each time the monitor threshold is hit, the monitor triggers a workflow run. 
+
+## Run history
+
+Once you trigger the workflow you can watch its progress and debug failed steps in the **Run History** view. Select an executed step to see the inputs, outputs, execution context, and error messages. The example below shows a step that failed due to an invalid Jira configuration. 
+
+{{< img src="/getting_started/workflow_automation/testing_the_workflow.mp4" alt="Preview of Workflow Testing" style="width:100%" video=true >}}
+
+To make edits to your workflow, click **Configuration**. To get back to the run history view, click **Run History**. 
+
+To view a list of previous workflow executions and whether each execution succeeded or failed, you can use the initial run history view. You can return to the initial execution history at any time by clicking on the workflow canvas. 
+
+## Conclusion
+
+You're done! When the monitor triggers the workflow, it creates a Jira issue for your engineering team to review. Here is an example Jira issue:
```

Comment:
```suggestion
When the monitor triggers the workflow, it creates a Jira issue for your engineering team to review. Here is an example Jira issue:
```
Sorry for removing the "You're done!" I do enjoy the friendly tone, but our team aims for a more informative tone in the docs.